backend/src/test/java/com/pickpick/message/application/ReminderServiceTest.java

```diff
@@ -98,6 +98,62 @@ private int findReminderSize(final Member member) {
         return reminderService.find(new ReminderFindRequest(null, null), member.getId()).getReminders().size();
     }
 
+    @DisplayName("리마인더 단건 조회")
+    @Test
+    void findOneReminder() {
+        // given
+        Long memberId = 2L;
+        Long messageId = 1L;
+
+        // when
+        ReminderResponse reminder = reminderService.findOne(messageId, memberId);
+
+        // then
+        assertAll(
+                () -> assertThat(reminder.getId()).isEqualTo(1L),
+                () -> assertThat(reminder.getRemindDate()).isEqualTo(LocalDateTime.of(2022, 8, 12, 14, 20))
+        );
+    }
+
+    @DisplayName("리마인더가 존재하지 않는 메시지를 단건 조회할 경우 예외 발생")
+    @Test
+    void findNotExistOneThenThrowException() {
+        // given
+        Long memberId = 2L;
+        Long messageId = 20L;
+
+        // when & then
+        assertThatThrownBy(() -> reminderService.findOne(messageId, memberId))
+                .isInstanceOf(ReminderNotFoundException.class);
+    }
+
+    @DisplayName("리마인더 목록 조회")
+    @ParameterizedTest(name = "{0}")
+    @MethodSource("parameterProvider")
+    void findReminders(final String subscription, final Long reminderId, final Long memberId,
+                       final List<Long> expectedIds, final boolean expectedIsLast) {
+        // given
+        given(clock.instant())
+                .willReturn(Instant.parse("2022-08-10T00:00:00Z"));
+
+        // when
+        ReminderResponses response = reminderService.find(new ReminderFindRequest(reminderId, null), memberId);
+
+        // then
+        List<Long> ids = convertToIds(response);
+        assertAll(
+                () -> assertThat(ids).containsExactlyElementsOf(expectedIds),
+                () -> assertThat(response.isLast()).isEqualTo(expectedIsLast)
+        );
```

Comment:
```suggestion
// when
ReminderResponses response = reminderService.find(new ReminderFindRequest(reminderId, null), memberId);
List<ReminderResponse> reminderResponses = response.getReminders();

// then
assertAll(
    () -> assertThat(reminderResponses).extracting("id").isEqualTo(expectedIds),
    () -> assertThat(response.isLast()).isEqualTo(expectedIsLast)
);
```

convertToIds 대신 extracting을 사용하면 어떨까요?